bindings/matrix-sdk-crypto-ffi/src/machine.rs

```diff
@@ -1461,12 +1495,15 @@ impl OlmMachine {
     ///     }
     /// }
     /// ```
-    pub fn verify_backup(&self, backup_info: &str) -> Result<bool, CryptoStoreError> {
+    pub fn verify_backup(
```

Comment:
Optional: I'd unify the `verify` and `check` terms in the api, e.g. have `SignatureVerification` as return value

Reply:
I hope I understood this correctly, did so in https://github.com/matrix-org/matrix-rust-sdk/pull/1217/commits/91bbaa09dc4947ed7a29b34e0540807212aa0fe9 as well.

Reply:
Yea thats exactly what I meant 👍